src/features/cookies/helpers/removeOptionalsCookiesKeys.test.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  KEYS_TO_REMOVE,
+  removeOptionalsCookiesKeys,
+} from 'features/cookies/helpers/removeOptionalsCookiesKeys'
+import { storage } from 'libs/storage'
+import { renderHook, superFlushWithAct } from 'tests/utils'
+
+describe('removeOptionalsCookiesKeys', () => {
+  it.each(KEYS_TO_REMOVE)(
+    'should not clear optional key "%s" if user has made choice less than 6 months ago',
```

Comment:
l'implem ne semble rien faire avec des dates

```suggestion
    'should not clear optional key "%s"',
```